src/User32.Desktop/User32.Helpers.cs

```diff
@@ -47,5 +47,24 @@ public static unsafe string GetClassName(IntPtr hWnd)
 
             return new string(className, 0, count);
         }
+
+        /// <summary>
+        /// Retrieves from the clipboard the name of the specified registered format.
+        /// </summary>
+        /// <param name = "format">The type of format to be retrieved. This parameter must not specify any of the predefined clipboard formats.</param>
+        /// <returns>The format name string.</returns>
+        /// <exception cref="Win32Exception">Thrown when an error occurs.</exception>
+        public static unsafe string GetClipboardFormatName(int format)
+        {
+            const int bufferSize = 256;
+            char* formatName = stackalloc char[bufferSize]; // max class name length
```

Comment:
Just fixed that. The default length is 255+NULL, thus 256 for both. Added an open-ended GetClassName helper getting the desired length because I could not find or infer anything about the maximum class name of a window.